openlibrary/catalog/add_book/load_book.py

```diff
@@ -279,7 +284,12 @@ def import_author(author: dict[str, Any], eastern=False) -> "Author | dict[str,
     :return: Open Library style Author representation, either existing Author with "key",
              or new candidate dict without "key".
     """
-    assert isinstance(author, dict)
+    assert isinstance(author, dict), "Author must be a dictionary."
+
+    # Validate that the author dictionary contains a valid name
+    if not author.get('name', '').strip():
+        raise ValueError("Author dictionary must contain a valid 'name' field.")
```

Comment:
This will prevent remote_ids ; 
```suggestion
    if not author.get('name', '').strip() and not author.get('remote_ids'):
        raise ValueError("Author dictionary must contain a valid 'name' or 'remote_ids' field.")
```